tests/unit/test_charm_2.py

```diff
@@ -215,3 +258,103 @@ def in_same_line(content, *matches):
                 ),
                 "db info in config should takes precedence over the db relation"
             )
+
+    def test_wp_install_cmd(self):
+        """
+        act: generate wp-cli command to install WordPress
+        assert: generated command should match current config and status
+        """
+        consensus = self._setup_replica_consensus()
```

Comment:
Would it be worth separating the arrange and assert blocks using an empty line to make this test easier to read and differentiate between executing behaviour and checking results? Similar comment for the rest of the tests

Reply:
I will add some empty lines between arrange and act/assert, thanks!